index.html

```diff
@@ -4,162 +4,89 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <meta name="author" content="Choco Li" />
-    <meta name="description" content="2 full page demonstration" />
-    <meta name="keywords" content="orchard, healthy, food" />
-    <meta name="Resource-type" content="Document" />
     <title>Orchard</title>
   </head>
-  <body>
-    <div id="fullpage">
-      <div class="section active" id="section0">
-        <section class="story">
-          <div class="content content--x-center">
-            <div class="imageBox">
-              <img
-                class="imageBox__image"
-                src="./img/crossField.png"
-                alt="a man carrying a child crossing a crop field"
-              />
+  <body class="bgBlack">
+    <div class="container">
+      <section class="row my-15 align-items-center">
+        <div class="px07 col-lg-4 col-md-6">
+          <img src="./img/crossField.png" alt="crossField" class="mw-100" />
+        </div>
+        <div class="px07 col-lg-4 col-md-6 d-flex flex-column">
```

Comment:
I am not familiar with bootstrap. Are `px071`, `textWhite`, and `textSpacing_04` belonging to bootstrap? If not, can you find the same usage from the Bootstrap? If you couldn't find one, can you keep the custom CSS naming consistent?

Reply:
Consistent with Bootstrap's naming?

Reply:
Sure, I agree you can have a different naming strategy for custom class names. However, things like `px071`  and `textWhite` should be available in Bootstrap

Reply:
I need `padding-left: 0.7rem, padding-right: 0.7rem` which can't be find in Bootstrap.
`text-white` will be used :)

Reply:
Consider to update `$spacer` so that you don't need to have your own set of CSS utilities class names? https://getbootstrap.com/docs/4.0/utilities/spacing/